tools/src/main/java/org/terracotta/utilities/io/FilesSupport.java

```diff
@@ -383,7 +383,7 @@ private void lockFile(Path file) {
       LOGGER.trace("Lock ended on \"{}\"", file);
     }
 
-    @SuppressFBWarnings("DLS_DEAD_LOCAL_STORE")
+    @SuppressFBWarnings({"DLS_DEAD_LOCAL_STORE", "RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE"})
```

Comment:
Dis you run into an issue here?

Reply:
Yes! With java 11 if I remember. Could it be possible that findbugs behaves differently depending on the compiler version ?

Of you grab my branch, edit it to remove this and push to your fork, it will build in your fork and you will see the findbugs error.

Reply:
Ok, I can accept there are differences -- Java 11 changed the bytecode expression of some things and that's been enough to throw off SpotBugs.

Reply:
The right way to avoid that would be to build our own docker images maintained with java 8 always plus some other jvm and a toolchain file so that we can always compile in 8 and run tests with the selected JVM. 

I think this is better to do like here because it also check the compilation with other version.. something that we never do at terracotta and we might have a bunch of surprises like that when we will have to compile in another version one day.